docs/guides/training_101/index.md

```diff
@@ -219,15 +219,14 @@ We will now define the configuration for the autoencoder. It holds the configura
 Example:
 
 ```python
-from refiners.training_utils import BaseConfig, TrainingConfig, OptimizerConfig, LRSchedulerConfig, Optimizers, LRSchedulers
-from refiners.training_utils.common import TimeUnit, TimeValue
+from refiners.training_utils import BaseConfig, TrainingConfig, OptimizerConfig, LRSchedulerConfig, Optimizers, LRSchedulers, Epoch
```

Comment:
`LRSchedulers` does not exist: it should be `LRSchedulerType` (can you also update it below (`lr_scheduler = LRSchedulerConfig(...)`)